yesod-core/src/Yesod/Core/Dispatch.hs

```diff
@@ -28,6 +28,7 @@ module Yesod.Core.Dispatch
     , setEqDerived
     , setShowDerived
     , setReadDerived
+    , setParameterisedSubroute
```

Comment:
```suggestion
    , setParameterizedSubroute
```

Nitpick I know but the repo *mostly* uses the `z` instead of `s` in this context